pkg/execution/state/pause.go

```diff
@@ -208,6 +208,9 @@ type Pause struct {
 	InvokeTargetFnID *string `json:"itFnID,omitempty"`
 	// SignalID is the ID of the signal that is responsible for this pause.
 	SignalID *string `json:"signalID,omitempty"`
+	// SupersedeSignalOnConflict indicates whether we should supersede the
+	// signal if a wait already exists for the signal ID.
+	SupersedeSignalOnConflict *bool `json:"-"`
```

Comment:
another absolute nit: probs doesnt need to be a ptr;  the default false is safe, and this then means we don't need to make a ptr to the opcode when handling this in the executor.  this is absolutely whatever, though, and really isn't blocking:  just may help prevent nil pointers accidentally in the future.